drake/core/test/matrix_compare_test.cc

```diff
@@ -0,0 +1,152 @@
+#include "drake/util/eigen_matrix_compare.h"
+#include "gtest/gtest.h"
+
+using drake::util::MatrixCompareType;
+
+namespace drake {
+namespace test {
```

Comment:
Nevermind. Since `eigen_matrix_compare_test.cc` specifically evaluates a function defined in `drake/util/eigen_matrix_compare.h`, I will move `eigen_matrix_compare_test.cc` into `drake/util/test/` and use:

```
namespace drake {
namespace util {
namespace test 
```